lib/active_record/connection_adapters/nulldb_adapter/table_definition.rb

```diff
@@ -1,5 +1,10 @@
 class ActiveRecord::ConnectionAdapters::NullDBAdapter
 
-  TableDefinition = ActiveRecord::ConnectionAdapters::TableDefinition
-
+  class TableDefinition < ActiveRecord::ConnectionAdapters::TableDefinition
+    attr_accessor :name
+    alias_method :citext, :text
+    alias_method :interval, :text
+    alias_method :geometry, :text
+    alias_method :jsonb, :json
```

Comment:
Some older versions of ActiveRecord do not define json columns, I believe this should be enough to make specs pass in other Ruby and ActiveRecord versions.

```suggestion
    alias_method :jsonb, :json if method_defined? :json
```